test/testtheme.cpp

```diff
@@ -0,0 +1,90 @@
+#include <QTest>
```

Comment:
You forgot the copyright text.

Yeah, I know, it sounds like having that in a header-only , is enough, but, in my past reviews with @er-vin, he insisted on having it in the source files as well.

Reply:
Yes, in some way one could argue this is even more important on the cpp than on the header. :-)

(header is "just" API while most of the IP related to business logic is in the cpp)